client/js/components/launchdarkly.js

```diff
@@ -0,0 +1,27 @@
+import LDClient from 'ldclient-js';
+import uuidV1 from 'uuid/v1';
+
+const LDUserKey = 'LDUserKey';
+
+let uuid = window.localStorage.getItem(LDUserKey);
+
+if (!uuid) {
+  uuid = uuidV1();
+  window.localStorage.setItem(LDUserKey, uuid);
+}
+
+const user = {
+  key: uuid,
+  anonymous: true
+};
+
+const ldclient = LDClient.initialize('58e63fe26b7519095c7484cb', user);
```

Comment:
Yep. From the docs, "Environment IDs are not secret-- they can be safely exposed in your client-side code."

http://docs.launchdarkly.com/v2.0/docs/js-sdk-reference